src/app/components/list Projects/style.module.scss

```diff
@@ -0,0 +1,28 @@
+$text-color: #64748B;
```

Comment:
i think no need for this variables as you use it once in the file and also in this file only!